docs/_docs/liquid/tags.md

```diff
@@ -57,15 +60,61 @@ numbers. For instance, the following code block would include line numbers next
 to each line:
 
 {% raw %}
+
 ```liquid
 {% highlight ruby linenos %}
 def foo
   puts 'foo'
 end
 {% endhighlight %}
 ```
+
+{% endraw %}
+
+### Variables for language and line numbers
+
+{: .note}
+To pass a language variable to the `highlight` tag, the variable **must** be named `lang`, as in {% raw %}{{ page.lang }}{% endraw %}. The `linenos` variable can be called by any other name.
+
+Both language and the line numbering parameter `linenos` can be passed to the `highlight` tag as variables, instead of specific paramters. \
```

Comment:
Add line break instead of \. And fix typo

```suggestion
Both the language and the line numbering parameter (`linenos`) can be passed to the `highlight` tag as variables, instead of the standard approach of fixed parameters.

```